client/packages/reports/src/ListView/SidePanel.tsx

```diff
@@ -70,7 +70,7 @@ export const SidePanelComponent = () => {
               label={t('label.monthly-consumption-look-back-period')}
               decimalLimit={2}
               endAdornment={t('label.months')}
-              value={preferences?.monthlyConsumptionLookBackPeriod}
+              value={preferences?.monthlyConsumptionLookBackPeriod !== 0.0 ? preferences?.monthlyConsumptionLookBackPeriod : preferences?.defaultMonthlyConsumptionLookBackPeriod}
```

Comment:
I think it's better to make monthlyConsumption not nullable return of preference with the default value, we shouldn't be handling defaults in front end, and as per before, let's make it easier on consumer